City-of-Helsinki/outdoors-sports-map
# Add Ice swimming places to map

## Description
 Display Ice swimming places to map as own types: 
  - provides ice swimming as outdoor sport choice
  - adds own icon for ice swimming place
  - displays condition, notice and other information on the detail of ice swimming place

## Context

[HULK-18](https://project.anders.fi/browse/HULK-18)

## How Has This Been Tested?

 Choose ice swimming from the sport options, check the places are displayed with the correct icons. Click on a place to see the detail, check if correct content is displayed with correct icons and details such as Condition, Notice, Information, Water temperature etc.

## Manual Testing Instructions for Reviewers

Choose ice swimming from the sport options, check the places are displayed with the correct icons. Click on a place to see the detail, check if correct content is displayed with correct icons and details such as Condition, Notice, Information, Water temperature etc.

## Screenshots
Sports choice:
![outdoor-sport-choice](https://user-images.githubusercontent.com/117433931/225302309-df9f386d-5a11-4fa7-a102-2ba7c88a8fb3.png)

Ice-swimming selected:
![Ice-swimming-selected-displayed-on-map](https://user-images.githubusercontent.com/117433931/225302430-091cb198-1376-4344-86a8-20777be08d75.png)

Single ice-swim place selected:
![single-ice-swim-place-selected-for-detail](https://user-images.githubusercontent.com/117433931/225302537-519d2e55-eb35-4133-a240-a1ec444368d2.png)
